packages/appium/lib/appium.js

```diff
@@ -522,9 +516,8 @@ class AppiumDriver extends DriverCore {
     // cases with plugin handling.
 
     const isGetStatus = cmd === GET_STATUS_COMMAND;
-    const isDeleteSession = cmd === DELETE_SESSION_COMMAND;
     const isUmbrellaCmd = !isGetStatus && isAppiumDriverCommand(cmd);
-    const isSessionCmd = !isUmbrellaCmd || isDeleteSession;
+    const isSessionCmd = isSessionCommand(cmd);
```

Comment:
does this evaluate to exactly the same value as the previous one? i remember having to tune this boolean to get plugin handling to work properly.

Reply:
according to the previous implementation it should be. Do you see any potential issues there?

Reply:
hmm the only thing I can think is that I introduced it in https://github.com/appium/appium/commit/7b3893a36202018de7c2124c2028bfbbd8a9d7fd. And maybe no one has tried to call GET /status without a session for a while? but this code has been published for longer than the report says the issue has existed, so i am not sure. maybe some interaction with this change and the driver refactor?

but yeah i think this change is good. i'm realizing we can also change the above line to simply:

```js
const isUmbrellaCmd = isAppiumDriverCommand(cmd);
```

because that function already returns true if it's not get status.